src/main/java/sparksoniq/ShellStart.java

```diff
@@ -25,6 +25,8 @@
 import sparksoniq.spark.SparkSessionManager;
 
 import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.Arrays;
 import java.util.HashMap;
```

Comment:
Remove unused imports (everything except java.io.IOExceptions and java.util.arrays)

Reply:
Done